Tests/ScenarioTester/ScenarioTesterTests/TestDataXScenarios.cs

```diff
@@ -0,0 +1,124 @@
+﻿using DataX.ServerScenarios;
```

Comment:
Header
// *********************************************************************
// Copyright (c) Microsoft Corporation.  All rights reserved.
// Licensed under the MIT License
// *********************************************************************